### PR TITLE
Release 37 merge

### PR DESCRIPTION
**Pull request by rprouse**
Merges the changes.txt back into master from the release and increments the version to 3.8 for ongoing development. @nunit/engine-team please review and merge.

### CODE REVIEW

File: CHANGES.txt [hunk 1]
```diff
@@ -1,4 +1,43 @@
-﻿NUnit 3.6.1 - March 6, 2017
+﻿NUnit 3.7 - July 13, 2017
```
Comment (jnm2) [inline CHANGES.txt]: I feel like this should say NUnit Console to disambiguate from NUnit Framework.

**Comment by jnm2** [comment]
@rprouse Is there a reason you requested a second review on this PR from me? I'm not seeing a new commit.

File: CHANGES.txt (hunk 1 above)
Comment (rprouse) [inline CHANGES.txt]: That would be a good idea. Unfortunately the release is out, but we could update it for the next release. I would like to switch this to a markdown file too, like we did for the framework.

**Comment by rprouse** [comment]
Nope, sorry, I didn't see that you approved. I had assumed you would just merge it.

**Comment by jnm2** [comment]
Okay. I had thought you wanted to wait for both requested reviews.

**Pull request merged**